src/Sylius/Bundle/CoreBundle/Calculator/PercentageDiscountPriceCalculator.php

```diff
@@ -18,9 +18,16 @@
 
 final class PercentageDiscountPriceCalculator implements ActionBasedPriceCalculatorInterface
 {
+    public const TYPE = 'percentage_discount';
+
+    public function getType(): string
```

Comment:
What do you think about making this field static? 
```suggestion
    public static function getType(): string
```